quipucords/scanner/network/inspect_callback.py

```diff
@@ -92,6 +93,8 @@ def _finalize_host(self, host):
         logger.debug('host scan complete for %s with facts %s',
                      host, facts)
 
+        results = initial_processing.process(facts)
```

Comment:
think we should move the above logging statement after this one a print results.